File: src/main/kotlin/dev/jefy/connectpro/engagement/applicaion/command/LikeCommand.java
package dev.jefy.connectpro.engagement.applicaion.command;

import org.jspecify.annotations.NullMarked;

/**
 * @author Jôph Yamba
 */
@NullMarked
public interface LikeCommand {

    void like(ServiceId serviceId);
    void unlike(ServiceId serviceId);
}

File: src/main/kotlin/dev/jefy/connectpro/engagement/application/command/LikeCommand.kt
package dev.jefy.connectpro.engagement.application.command

import dev.jefy.connectpro.portfolio.domain.vo.ServiceId

/**
 * @author Jôph Yamba
 */

interface LikeCommand {
    fun like(serviceId: ServiceId)
    fun unlike(serviceId: ServiceId)
}

